cli/README.md

```diff
@@ -0,0 +1,46 @@
+# AMP-Toolbox CLI
+
+A command line interface (CLI) for AMP-Toolbox
+
+Supported commands:
+- [update-cache](#update-cache): CLI for the [AMP update-cache](https://developers.google.com/amp/cache/update-cache).
+- [version](#version): Prints the current version.
```

Comment:
suggestion: `prints the current AMP runtime version.`

Reply:
Actually, this prints the CLI version.